src/methods/geneformer/script.py

```diff
@@ -11,74 +15,74 @@
 
 ## VIASH START
 par = {
-  'input_train': 'resources_test/task_label_projection/cxg_immune_cell_atlas/train.h5ad',
-  'input_test': 'resources_test/task_label_projection/cxg_immune_cell_atlas/test.h5ad',
-  'output': 'output.h5ad',
-  "model": "gf-12L-95M-i4096",
-}
-meta = {
-  'name': 'geneformer'
+    "input_train": "resources_test/task_label_projection/cxg_immune_cell_atlas/train.h5ad",
+    "input_test": "resources_test/task_label_projection/cxg_immune_cell_atlas/test.h5ad",
+    "output": "output.h5ad",
+    "model": "gf-12L-95M-i4096",
 }
+meta = {"name": "geneformer"}
 ## VIASH END
 
 n_processors = os.cpu_count()
 
 sys.path.append(meta["resources_dir"])
 from exit_codes import exit_non_applicable
 
-print('>>> Reading input files', flush=True)
-input_train = ad.read_h5ad(par['input_train'])
-input_test = ad.read_h5ad(par['input_test'])
+print(">>> Reading input files", flush=True)
+input_train = ad.read_h5ad(par["input_train"])
+input_test = ad.read_h5ad(par["input_test"])
 
 if input_train.uns["dataset_organism"] != "homo_sapiens":
-  exit_non_applicable(
-    f"Geneformer can only be used with human data "
-    f"(dataset_organism == '{input_train.uns['dataset_organism']}')"
-  )
+    exit_non_applicable(
+        f"Geneformer can only be used with human data "
+        f"(dataset_organism == '{input_train.uns['dataset_organism']}')"
+    )
 
 is_ensembl = all(var_name.startswith("ENSG") for var_name in input_train.var_names)
```

Comment:
Maybe it makes sense to set set `adata.var_names = adata.var["feature_id"]` here first?